common/djangoapps/student/management/commands/set_superuser.py

```diff
@@ -0,0 +1,45 @@
+"""Management command to grant or revoke superuser access for one or more users"""
+
+from optparse import make_option
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """Management command to grant or revoke superuser access for one or more users"""
+    option_list = BaseCommand.option_list + (
+        make_option('--unset',
+                    action='store_true',
+                    dest='unset',
+                    default=False,
+                    help='Set is_superuser to False instead of True'),
+    )
+
+    args = '<user|email> [user|email ...]>'
+    help = """
+    This command will set is_superuser to true for one or more users.
+    Lookup by username or email address, assumes usernames
+    do not look like email addresses.
+    """
+
+    def handle(self, *args, **options):
+        if len(args) < 1:
+            raise CommandError('Usage is set_superuser {0}'.format(self.args))
+
+        for user in args:
+            try:
+                if '@' in user:
+                    userobj = User.objects.get(email=user)
+                else:
+                    userobj = User.objects.get(username=user)
+            except Exception as e:  # pylint: disable=broad-except
+                print "Error geting user with identifier {}: {}: {}".format(user, type(e).__name__, e.message)
```

Comment:
@maxrothman you either need to move the `except` to after the `userobj.save()` call, or add a `continue` statement here.  Otherwise, after this print statement, the loop will either continue with the value of `userobj` potentially pointing to the user from a previous iteration, or not defined at all, which will cause a NameError.